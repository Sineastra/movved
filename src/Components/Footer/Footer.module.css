.footer {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  align-items: initial;
  align-content: initial;
  padding-left: 20px;
  padding-right: 20px;
  font-family: "Nunito", sans-serif;
  color: var(--theme-text-color);
  background-color: var(--theme-main-color_1);
  padding-top: 30px;
}
.footer ul {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  align-items: initial;
  align-content: initial;
  height: 170px;
  padding-left: 0;
  color: var(--theme-subtext-color);
}
.footer ul > li {
  list-style: none;
}

.socialIcons {
  color: var(--theme-subtext-color);
  font-size: 30px;
  margin-right: 25px;
}
.socialIcons:hover {
  color: var(--theme-text-color);
}

.socialSection {
  color: var(--theme-subtext-color);
  display: flex;
  flex-direction: row;
  justify-content: initial;
  align-items: center;
  align-content: initial;
  border-bottom: 1px solid #1f1f1f;
  padding-bottom: 25px;
  margin-bottom: 10px;
}
.socialSection > h3 {
  margin: 0 30px 0 0;
}

/*# sourceMappingURL=Footer.module.css.map */
