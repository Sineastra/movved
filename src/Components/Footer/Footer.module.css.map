{"version":3,"sourceRoot":"","sources":["Footer.module.scss","../../_scssPartials/mixins/_mixins.scss","../../_scssPartials/mixins/_mediaQueries.scss"],"names":[],"mappings":"AAGA;EACI;EACA;EACA;EACA;EACA;EACA;ECRA;EACA,gBDQqB;ECPrB,iBDO6B;ECN7B,aAJgE;EAKhE,eALwF;;ADYxF;EACI;EACA;EACA;ECdJ;EACA,gBDcyB;ECbzB,iBDaiC;ECZjC,aAJgE;EAKhE,eALwF;;ADkBpF;EACI;;AAIR;EACI;EACA;;;AAIR;EACI;EACA;EACA;;AAEA;EACI;;AElCJ;EF4BJ;IAUQ;IACA;;;;AAIR;EACI;EACA;EACA;EACA;EC/CA;EACA,gBD+CqB;EC9CrB,iBD8C0B;EC7C1B,aD6CmC;EC5CnC,eALwF;;ADmDxF;EACI;;AE9CJ;EF6CA;IAIQ;;;AEtDR;EFkDA;IAQQ;;;AErDR;EFsCJ;IAoBQ;IACA;IChEJ;IACA,gBDgEyB;IC/DzB,iBAH0C;IAI1C,aAJgE;IAKhE,eALwF;;;;ADsE5F;EACI;EACA;;AAEA;EACI;EACA;EACA;EACA;;;AAIR;ECjFI;EACA,gBAM4B;EAL5B,iBAMiC;EALjC,aAKyC;EAJzC,eAIiD;ED2EjD;;AAEA;EACI;EACA","file":"Footer.module.css"}