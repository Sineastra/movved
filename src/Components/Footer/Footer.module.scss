@import "../../_scssPartials/mixins/mixins";

.footer {
    padding-top: 30px;
    padding-left: 20px;
    padding-right: 20px;
    color: var(--theme-text-color);
    font-family: 'Nunito', sans-serif;
    background-color: var(--theme-main-color_1);
    @include displayFlex(column, space-between);

    & ul {
        height: 170px;
        padding-left: 0;
        color: var(--theme-subtext-color);
        @include displayFlex(column, space-between);

        & > li {
            list-style: none;
        }
    }

    & a {
        text-decoration: none;
        color: var(--theme-subtext-color);
    }
}

.socialIcons {
    font-size: 30px;
    margin-right: 25px;
    color: var(--theme-subtext-color);

    &:hover {
        color: var(--theme-text-color);
    }
}

.socialSection {
    color: var(--theme-subtext-color);
    @include displayFlex(row, initial, center);

    & > h3 {
        margin: 0 30px 0 0;
    }

    padding-bottom: 25px;
    margin-bottom: 10px;
    border-bottom: 1px solid #1f1f1f;
}

.mainLogoSection {
    margin-top: 30px;
    margin-bottom: 20px;

    & > span {
        font-family: 'Nunito', sans-serif;
        color: var(--theme-subtext-color);
        font-size: 0.8rem;
        font-style: italic;
    }
}

.mainLogoWrapper {
    @include centeredFlex();
    border-top: 1px solid #1f1f1f;

    & > a {
        color: var(--theme-secondary-color);

        width: 150px;
    }
}