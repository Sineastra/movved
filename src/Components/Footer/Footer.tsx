import styles from "./Footer.module.scss"
import { FaFacebook, FaInstagram, FaLinkedinIn, FaTwitter } from "react-icons/fa"
import { IconContext } from "react-icons"
import { Link } from "react-router-dom"
import MainLogoSVG from "../_SVGs/MainLogoSVG"


const Footer = () => {

	return (
		<footer className={ styles.footer }>
			<section className={ styles.socialSection }>
				<h3>Последвайте ни: </h3>
				<IconContext.Provider value={ { className: styles.socialIcons } }>
					<a href="https://facebook.com"><FaFacebook/></a>
					<a href="https://instagram.com"><FaInstagram/></a>
					<a href="https://twitter.com"><FaTwitter/></a>
					<a href="https://linkedin.com"><FaLinkedinIn/></a>
				</IconContext.Provider>
			</section>
			<section>
				<h3>Избрани жанрове</h3>
				<ul>
					<li>
						<Link to="/movies/action">
							Екшън Филми
						</Link>
					</li>
					<li>
						<Link to="/series">
							Сериали
						</Link>
					</li>
					<li>
						<Link to="/movies/animation">
							Анимации
						</Link>
					</li>
					<li>
						<Link to="/movies/family">
							За Семейството
						</Link>
					</li>
					<li>
						<Link to="/movies/horror">
							Страшни Филми
						</Link>
					</li>
					<li>
						<Link to="/movies/documentary">
							Документални
						</Link>
					</li>
				</ul>
			</section>
			<section>
				<h3>Още раздели</h3>
				<ul>
					<li>
						<Link to="/movies">
							Всички филми
						</Link>
					</li>
					<li>
						<Link to={ "/movies/random" }>
							Случаен филм
						</Link>
					</li>
					<li>
						<Link to={ "/movies/popular" }>
							Популярни филми
						</Link>
					</li>
					<li>
						<Link to={ "/movies/bg-audio" }>
							Филми с БГ аудио
						</Link>
					</li>
					<li>
						<Link to="/">
							Lorem ipsum dolor sit amet.
						</Link>
					</li>
					<li>
						<Link to="/">
							Lorem ipsum dolor sit amet.
						</Link>
					</li>
				</ul>
			</section>
			<section>
				<h3>За нас</h3>
				<ul>
					<li>
						<a href="https://facebook.com">Facebook</a>
					</li>
					<li>
						<a href="https://instagram.com">Instagram</a>
					</li>
					<li>
						<a href="https://twitter.com">Twitter</a>
					</li>
					<li>
						<a href="https://linkedin.com">LinkedIn</a>
					</li>
					<li>
						<a href="https://zamunda.net">Zamunda</a>
					</li>
					<li>
						<a href="https://apogee99.com/index.php?main_page=index&cPath=6_324">Щепсели (Контакти)</a>
					</li>
				</ul>
			</section>
			<section className={ styles.mainLogoSection }>
				<div className={ styles.mainLogoWrapper }>
					<Link to="/">
						<MainLogoSVG/>
					</Link>
				</div>
				<span>
				Този сайт не съхранява никакви физически видео файлове на сървъра си. Всички видеа са прикачени от други
				уебсайтове, с които не сме обвързани.
				</span>
			</section>
		</footer>
	)
}

export default Footer