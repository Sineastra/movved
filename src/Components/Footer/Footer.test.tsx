import renderer from "react-test-renderer"
import { BrowserRouter } from "react-router-dom"
import Footer from "./Footer"


describe("---> Testing the footer (it is static, so only a snapshot test", () => {
	it("snapshots should match", () => {
		const tree = renderer.create(
			<BrowserRouter>
				<Footer/>
			</BrowserRouter>,
		).toJSON()

		expect(tree).toMatchSnapshot()
	})
})