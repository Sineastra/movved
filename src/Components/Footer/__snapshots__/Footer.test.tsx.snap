// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`---> Testing the footer (it is static, so only a snapshot test snapshots should match 1`] = `
<footer
  className="footer"
>
  <section
    className="socialSection"
  >
    <h3>
      Последвайте ни: 
    </h3>
    <a
      href="https://facebook.com"
    >
      <svg
        className="socialIcons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
        />
      </svg>
    </a>
    <a
      href="https://instagram.com"
    >
      <svg
        className="socialIcons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
        />
      </svg>
    </a>
    <a
      href="https://twitter.com"
    >
      <svg
        className="socialIcons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
        />
      </svg>
    </a>
    <a
      href="https://linkedin.com"
    >
      <svg
        className="socialIcons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
        />
      </svg>
    </a>
  </section>
  <section>
    <h3>
      Избрани жанрове
    </h3>
    <ul>
      <li>
        <a
          href="/movies/action"
          onClick={[Function]}
        >
          Екшън Филми
        </a>
      </li>
      <li>
        <a
          href="/series"
          onClick={[Function]}
        >
          Сериали
        </a>
      </li>
      <li>
        <a
          href="/movies/animation"
          onClick={[Function]}
        >
          Анимации
        </a>
      </li>
      <li>
        <a
          href="/movies/family"
          onClick={[Function]}
        >
          За Семейството
        </a>
      </li>
      <li>
        <a
          href="/movies/horror"
          onClick={[Function]}
        >
          Страшни Филми
        </a>
      </li>
      <li>
        <a
          href="/movies/documentary"
          onClick={[Function]}
        >
          Документални
        </a>
      </li>
    </ul>
  </section>
  <section>
    <h3>
      Още раздели
    </h3>
    <ul>
      <li>
        <a
          href="/movies"
          onClick={[Function]}
        >
          Всички филми
        </a>
      </li>
      <li>
        <a
          href="/movies/random"
          onClick={[Function]}
        >
          Случаен филм
        </a>
      </li>
      <li>
        <a
          href="/movies/popular"
          onClick={[Function]}
        >
          Популярни филми
        </a>
      </li>
      <li>
        <a
          href="/movies/bg-audio"
          onClick={[Function]}
        >
          Филми с БГ аудио
        </a>
      </li>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Lorem ipsum dolor sit amet.
        </a>
      </li>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Lorem ipsum dolor sit amet.
        </a>
      </li>
    </ul>
  </section>
  <section>
    <h3>
      За нас
    </h3>
    <ul>
      <li>
        <a
          href="https://facebook.com"
        >
          Facebook
        </a>
      </li>
      <li>
        <a
          href="https://instagram.com"
        >
          Instagram
        </a>
      </li>
      <li>
        <a
          href="https://twitter.com"
        >
          Twitter
        </a>
      </li>
      <li>
        <a
          href="https://linkedin.com"
        >
          LinkedIn
        </a>
      </li>
      <li>
        <a
          href="https://zamunda.net"
        >
          Zamunda
        </a>
      </li>
      <li>
        <a
          href="https://apogee99.com/index.php?main_page=index&cPath=6_324"
        >
          Щепсели (Контакти)
        </a>
      </li>
    </ul>
  </section>
  <section
    className="mainLogoSection"
  >
    <div
      className="mainLogoWrapper"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <svg
          enableBackground="new 0 0 1000 1000"
          role="main-logo"
          version="1.1"
          viewBox="0 0 1000 1000"
          x="0px"
          y="0px"
        >
          <metadata>
             Svg Vector Icons : http://www.onlinewebfonts.com/icon
          </metadata>
          <g
            fill="currentcolor"
          >
            <path
              d="M820.5,587.9"
            />
            <path
              d="M944.1,285.6H55.9c-25.4,0-45.9,20.6-45.9,45.9v336.9c0,25.4,20.6,45.9,45.9,45.9h888.1c25.4,0,45.9-20.6,45.9-45.9V331.6C990,306.2,969.4,285.6,944.1,285.6z M265.9,515l-112.6,92.7c-10,8.3-18.3,4.4-18.3-8.6V400.9c0-13,8.2-16.9,18.3-8.6L265.9,485C276,493.2,276,506.8,265.9,515z M959.4,658c0,14.2-10.6,25.7-23.7,25.7H408.5c-13,0-23.7-11.6-23.7-25.7V342c0-14.2,10.6-25.7,23.7-25.7h527.2c13,0,23.7,11.6,23.7,25.7V658z"
            />
            <path
              d="M697.3,392.4c-2.3-9.1-10.2-15.6-19.6-16.1c-9.4-0.5-18,5.1-21.3,13.8l-57.2,151.6l-57.2-151.6c-3.3-8.7-12-14.4-21.3-13.8c-9.4,0.5-17.3,7.1-19.6,16.1l-51.6,204.9c-2.9,11.4,4.1,23,15.6,25.9c11.5,2.9,23.2-4.1,26.1-15.5l34.8-138.4l53.1,140.7c3.1,8.3,11.2,13.8,20.1,13.8c9,0,17-5.5,20.1-13.8l53.1-140.7l34.8,138.4c2.4,9.7,11.2,16.2,20.8,16.2c1.7,0,3.5-0.2,5.2-0.6c11.5-2.9,18.5-14.4,15.6-25.9L697.3,392.4z"
            />
            <path
              d="M879.3,376.9c-11.5-2.9-23.2,4.1-26.1,15.5l-30.7,122.1l-30.7-122.1c-2.9-11.4-14.5-18.4-26.1-15.5c-11.5,2.9-18.5,14.4-15.6,25.9l51.6,204.9c2.4,9.5,11,16.2,20.8,16.2c9.9,0,18.5-6.7,20.8-16.2L895,402.7C897.8,391.3,890.8,379.7,879.3,376.9z"
            />
          </g>
        </svg>
      </a>
    </div>
    <span>
      Този сайт не съхранява никакви физически видео файлове на сървъра си. Всички видеа са прикачени от други уебсайтове, с които не сме обвързани.
    </span>
  </section>
</footer>
`;
